editor/assets/chunks/common/shadow/native-pcf.chunk

```diff
@@ -4,13 +4,20 @@
 #pragma define CC_SHADOW_PCF_SOFT 1
 #pragma define CC_SHADOW_PCF_SOFT_3X 2
 #pragma define CC_SHADOW_PCF_SOFT_5X 3
-
+#pragma extension([GL_EXT_shader_texture_lod, __VERSION__ < 130])
+vec4 shadowTexure(highp sampler2D shadowMap, vec2 coord) {
+    #if defined(CC_USE_WGPU)
+        return textureLod(shadowMap, coord, 0.0);
```

Comment:
Why textureLod is used on WGPU?

Reply:
texture doesn't have mips and these need to change to textureSampleLevel

Reply:
Otherwise, an error will be reported: 'textureSample' must only be called from uniform control flow

Reply:
I see. We might add the potential error message to the comments.
This uniform control flow requirement of 'textureSample' is WGSL only, people might not be aware of this.